misc/ls-extensions/modules/json-to-record-converter/build.gradle

```diff
@@ -36,6 +36,7 @@ dependencies {
         exclude group: "javax.validation", module: "validation-api"
         exclude group: 'com.google.guava', module: 'guava'
     }
+    implementation 'com.google.guava:guava:30.0-jre' // Required during runtime
```

Comment:
Can we refer to this module version from properties file? "${project.guavaVersion}"